### PR TITLE
List installed packages with "pip list"

### PR DESCRIPTION
**Pull request by JCristobal**

### CODE REVIEW

File: docs/dev/virtualenvs.rst [hunk 1]
```diff
@@ -100,9 +100,10 @@ the current state of the environment packages. To do this, run
 
 This will create a :file:`requirements.txt` file, which contains a simple
 list of all the packages in the current environment, and their respective
-versions. Later it will be easier for a different developer (or you, if you
-need to re-create the environment) to install the same packages using the
-same versions:
+versions. You can see the list of installed packages withot the requirements
```
Comment (sigmavirus24) [inline docs/dev/virtualenvs.rst]: Did you mean "without" instead of "withot"?

File: docs/dev/virtualenvs.rst [hunk 2]
```diff
@@ -100,9 +100,10 @@ the current state of the environment packages. To do this, run
 
 This will create a :file:`requirements.txt` file, which contains a simple
 list of all the packages in the current environment, and their respective
-versions. Later it will be easier for a different developer (or you, if you
-need to re-create the environment) to install the same packages using the
-same versions:
+versions. You can see the list of installed packages withot the requirements 
+format using "pip list". Later it will be easier for a different developer 
+(or you, if youneed to re-create the environment) to install the same packages
```
Comment (sigmavirus24) [inline docs/dev/virtualenvs.rst]: "you" and "need" should be separate words.

**Comment by sigmavirus24** [comment]
@JCristobal when you update this ping me.

**Comment by JCristobal** [comment]
Bug fixed!

https://github.com/kennethreitz/python-guide/pull/624

Thanks!

2015-10-23 17:19 GMT+02:00 Ian Cordasco <notifications@github.com>:

**Comment by sigmavirus24** [comment]
@JCristobal would you mind squashing your two commits? The latter has a misspelled and vague message.

**Comment by sigmavirus24** [comment]
Merged in 0f6372f6a0a33305dcb60538b417da00ad2a3e5c

**Pull request closed without merging**